File: tests/fixtures/multi_lang_sample/Sample.java
// Simple Java test file
import java.util.List;
import java.util.ArrayList;

public class Person {
    private String name;
    private int age;
    
    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }
    
    public String getName() {
        return this.name;
    }
    
    public void setName(String name) {
        this.name = name;
    }
    
    public int getAge() {
        return this.age;
    }
}

class Helper {
    public static void printMessage(String message) {
        System.out.println(message);
    }
}

File: tests/fixtures/multi_lang_sample/sample.cpp
// Simple C++ test file
#include <iostream>
#include <string>

class Person {
private:
    std::string name;
    int age;
    
public:
    Person(std::string n, int a) : name(n), age(a) {}
    
    std::string getName() {
        return name;
    }
    
    void setName(std::string n) {
        name = n;
    }
    
    int getAge() {
        return age;
    }
};

void printMessage(std::string message) {
    std::cout << message << std::endl;
}

int add(int a, int b) {
    return a + b;
}
